src/zui/components/ZUIButton/index.tsx

```diff
@@ -215,14 +215,15 @@ const ZUIButton: FC<ZUIButtonProps> = ({
               backgroundColor: theme.palette.warning.dark,
             },
             '&.MuiButton-outlinedPrimary': {
-              backgroundColor: theme.palette.grey[100],
+              backgroundColor: alpha(theme.palette.primary.main, 0.11),
             },
             '&.MuiButton-textPrimary': {
-              backgroundColor: theme.palette.grey[100],
+              backgroundColor: alpha(theme.palette.primary.main, 0.11),
```

Comment:
Just curious why we use the alpha() function from MUI here, I guess we don't have a more similar grey in our Palette?